cluster/manifests/02-kube-aws-iam-controller/deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       hostNetwork: true
       containers:
       - name: kube-aws-iam-controller
-        image: container-registry.zalando.net/teapot/kube-aws-iam-controller:v0.2.0
+        image: registry.opensource.zalan.do/teapot/kube-aws-iam-controller:v0.2.0-3-g8b47d5a
```

Comment:
This should still be `container-registry.zalando.net` or we can't run this on ARM.

Reply:
Oh, good catch. Fixed.